recon_test_pack/run_test_listmode_recon.sh

```diff
@@ -0,0 +1,141 @@
+#! /bin/sh
+# A script to check to see if reconstruction of simulated data gives the expected result.
+#
+#  Copyright (C) 2011 - 2011-01-14, Hammersmith Imanet Ltd
+#  Copyright (C) 2011-07-01 - 2011, Kris Thielemans
+#  Copyright (C) 2014, University College London
+#  This file is part of STIR.
+#
+#  This file is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+#  This file is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  See STIR/LICENSE.txt for details
+#      
+# Author Nikos Efthimiou
+#
+
+# Scripts should exit with error code when a test fails:
+if [ -n "$TRAVIS" ]; then
+    # The code runs inside Travis
+    set -e
+fi
+
+echo This script should work with STIR version ">"3.0. If you have
+echo a later version, you might have to update your test pack.
+echo Please check the web site.
+echo
+
+#
+# Options
+#
+MPIRUN=""
+
+#
+# Parse option arguments (--)
+# Note that the -- is required to suppress interpretation of $1 as options 
+# to expr
+#
+while test `expr -- "$1" : "--.*"` -gt 0
+do
+
+  if test "$1" = "--mpicmd"
+  then
+    MPIRUN="$2"
+    shift 1
+  elif test "$1" = "--help"
+  then
+    echo "Usage: `basename $0` [--mpicmd somecmd] [install_dir]"
+    echo "(where [] means that an argument is optional)"
+    echo "See README.txt for more info."
+    exit 1
+  else
+    echo Warning: Unknown option "$1"
+    echo rerun with --help for more info.
+    exit 1
+  fi
+
+  shift 1
+
+done 
+
+if [ $# -eq 1 ]; then
+  echo "Prepending $1 to your PATH for the duration of this script."
+  PATH=$1:$PATH
+fi
+
+# first delete any files remaining from a previous run
+rm -f my_*v my_*s my_*S
+
+echo "=== calculate background data"
+echo "===== randoms"
+
+echo "====== create delayed fansums"
+INPUT=PET_ACQ_small.l.hdr.STIR OUTPUT=my_fansums_delayed ${INSTALL_DIR}lm_fansums lm_fansums_delayed.par 2>my_fansums.log
```

Comment:
I'm in 2 minds about this. it would be good to include `lm_fansums` etc in a test. However, here it is irrelevant whatever background we use. So we create a dependency of the text on something what it isn't supposed to test. what do you think?

Reply:
I see your point. Lets raise a task on this but we can keep it for the time being.

Reply:
ok